site/content/en/docs/tasks/run/multikueue/kuberay.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: "Run Kuberay Jobs in Multi-Cluster"
-linkTitle: "Kuberay"
+title: "Run KubeRay Jobs in Multi-Cluster"
```

Comment:
Are all other places where KubeRay is mentioned in the correct case?

Reply:
Almost, yes. But for KEP and implementations, we still use `Kuberay` somewhere.